ggshield/scan/docker.py

```diff
@@ -134,6 +136,22 @@ def _get_layers_files(
         yield from _get_layer_files(archive, layer_info)
 
 
+def _validate_filepath(
+    filepath: str,
+    extension_banlist: Iterable[str],
+    filepath_banlist_patterns: Iterable[re.Pattern],
```

Comment:
nit: since this function is always called with `DEFAULT_EXTENSION_BANLIST` and `FILEPATH_BANLIST_PATTERNS` as arguments, maybe these two arguments can be removed?

Reply:
I thought we may change the banlists at some point, but I think you are right. 👍